governance/votes/Executive Vote - May 17, 2023.md

```diff
@@ -0,0 +1,88 @@
+* 23.8 MKR will be transferred to [Frontier Research](https://forum.makerdao.com/t/frontier-research-ad-communications/20441) at [0xA2d55b89654079987CF3985aEff5A7Bd44DA15A8](https://etherscan.io/address/0xA2d55b89654079987CF3985aEff5A7Bd44DA15A8).
+* 23.8 MKR will be transferred to [GFX Labs](https://forum.makerdao.com/t/constitutional-delegate-recognition-submission/20407) at [0x9B68c14e936104e9a7a24c712BEecdc220002984](https://etherscan.io/address/0x9B68c14e936104e9a7a24c712BEecdc220002984).
+* 23.8 MKR will be transferred to [QGov](https://forum.makerdao.com/t/qgov-ad-recognition-submission/20494) at [0xB0524D8707F76c681901b782372EbeD2d4bA28a6](https://etherscan.io/address/0xB0524D8707F76c681901b782372EbeD2d4bA28a6).
+* 23.8 MKR will be transferred to [TRUE NAME](https://forum.makerdao.com/t/true-name-aligned-delegate-communications-platform/20455) at [0x612F7924c367575a0Edf21333D96b15F1B345A5d](https://etherscan.io/address/0x612F7924c367575a0Edf21333D96b15F1B345A5d).
+* 23.8 MKR will be transferred to [vigilant](https://forum.makerdao.com/t/cd-recognition-submission-vigilant/20457) at [0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61).
+* 5.95 MKR will be transferred to [CodeKnight](https://forum.makerdao.com/t/codeknight-constitutional-delegate-communications/20443) at [0xf6006d4cF95d6CB2CD1E24AC215D5BF3bca81e7D](https://etherscan.io/address/0xf6006d4cF95d6CB2CD1E24AC215D5BF3bca81e7D).
+* 5.95 MKR will be transferred to [Flip Flop Flap Delegate LLC](https://forum.makerdao.com/t/flip-flop-flap-delegate-llc-aligned-delegate-recognition-submission-and-communication/20477) at [0x3d9751EFd857662f2B007A881e05CfD1D7833484](https://etherscan.io/address/0x3d9751EFd857662f2B007A881e05CfD1D7833484).
+* 5.95 MKR will be transferred to [PBG](https://forum.makerdao.com/t/pbg-aligned-delegate-communication-platform/20471) at [0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2).
+* 5.95 MKR will be transferred to [UPMaker](https://forum.makerdao.com/t/upmaker-ad-submission/20577) at [0xbB819DF169670DC71A16F58F55956FE642cc6BcD](https://etherscan.io/address/0xbB819DF169670DC71A16F58F55956FE642cc6BcD).
+
+### DAI Budget Streams
+
+As per [MIP107](https://mips.makerdao.com/mips/details/MIP107), the following DAI budget streams will be created if this executive proposal passes:
+
+#### DAI Budget Streams
+
+| Budget | DAI Amount | Start Date | End Date | Destination Address |
+|---|---|---|---|---|
+| Governance Security Engineering | 2,200,000 | 2023-05-01 | 2024-05-01 (--> $check date since it's midnight <---) | [0x569fAD613887ddd8c1815b56A00005BCA7FDa9C0](https://etherscan.io/address/0x569fAD613887ddd8c1815b56A00005BCA7FDa9C0) (--> $source of truth for this address other than spreadsheet? <---) |
```

Comment:
Source of truth is the spreadsheet. Retro should be posting something you can link to as well.

Reply:
I don't see a post yet so link to be added.